modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// *
+:_module-type: PROCEDURE
+[id="ipi-verifying-nodes-after-installation_{context}"]
+= Verifying node state after installation
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
+Your OpenShift Container Platform installation completes when the following installation health checks are successful:
+
+* The provisioning host can access the OpenShift Web Console.
+
+* All master nodes are ready.
+
+* All cluster Operators are available.
+
+However, although your installation successfully completes, not all worker nodes in the cluster are instantly provisioned. Some worker nodes remain in a non-provisioned state, such as NotReady, Inspecting or Provisioned. 
+
+The cluster Operator nodes continuously attempt to provision worker nodes for a set period of time. The state of a worker node only changes to degraded when the cluster Operator node fails to provision the worker node within this time period. You can define this time period in the install-config.yaml(???) file.
```

Comment:
remove this line, include it in note above.

Reply:
Done